app/src/main/java/com/example/touchcar/presentation/choose_body/recycler/ChooseBodyAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package com.example.touchcar.presentation.choose_body.recycler
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.RecyclerView
+import com.example.touchcar.databinding.MarketRecyclerItemBinding
+import com.example.touchcar.domain.entity.Body
+import com.example.touchcar.domain.entity.Market
+
+class ChooseBodyAdapter(private val onItemClickListener: (Body) -> Unit) :
+    RecyclerView.Adapter<ChooseBodyViewHolder>() {
```

Comment:
Давай договоримся, что если супер-класс переносим на следующую строку, то переносим и `) :`
```suggestion
class ChooseBodyAdapter(
    private val onItemClickListener: (Body) -> Unit
) : RecyclerView.Adapter<ChooseBodyViewHolder>() {
```